defaults/main.yml

```diff
@@ -13,14 +13,12 @@ karaf_log_path: ${karaf.data}/log/
 
 karaf_logging_template: org.ops4j.pax.logging.cfg
 
-karaf_java_home: "{% if ansible_os_family == 'RedHat' %}/usr/lib/jvm/java-1.8.0-openjdk/{% else %}/usr/lib/jvm/java-8-openjdk-amd64{% endif %}"
+karaf_java_home: /usr/lib/jvm/java-8-openjdk-amd64
```

Comment:
Should all these variables be removed as they are now set in the various `vars` files?

Reply:
You are right, since whatever value the have in defaults will get overridden by the vars value due to the include. (I think.)